packages/frontend/src/components/DashboardTabs/index.tsx

```diff
@@ -192,6 +192,9 @@ const DashboardTabs: FC<DashboardTabsProps> = ({
                     order: 0,
                 };
                 newTabs.push(firstTab);
+                dashboardTiles?.forEach((tile) => {
+                    tile.tabUuid = firstTab.uuid; // move all tiles to default tab
```

Comment:
I tried deleting a tab with charts, but they were not copied to the first tab.  See recording

[Screencast from 23-04-24 11:12:11.webm](https://github.com/lightdash/lightdash/assets/1983672/164c6c70-64c8-4833-b0b6-998bc41b4d43)

Reply:
sorry this comment might be misleading... i mean when you create the tab for the first time, this line will update all tiles' tabUuid to use default tab uuid, like they are 'moved' to default tab.

the video is the expected behaviours, when you delete the tab, all tiles in this tab will be removed as well.
unless the tab is the last one, then it will keep all the tiles.